channeldb/channel.go

```diff
@@ -2111,6 +2111,11 @@ func (c *OpenChannel) AppendRemoteCommitChain(diff *CommitDiff) error {
 			return ErrChanBorked
 		}
 
+		// Delete the contents of unsignedAckedUpdatesKey.
```

Comment:
should expand comment with why this can be deleted now.

Reply:
Done